sync/trypush.py

```diff
@@ -400,7 +400,14 @@ def included(t):
             # if a name is on the excluded list, only download SUCCESS job logs
             name = t.get("task", {}).get("metadata", {}).get("name")
             state = t.get("status", {}).get("state")
-            return name not in exclude or state == tc.SUCCESS
+            output = name not in exclude or state == tc.SUCCESS
+
+            # Add this name to exclusion list, so that we don't download the repeated run
+            # logs in a stability try run
+            if stability and name is not None and name not in exclude:
```

Comment:
I don't understand why you are excluding the repeated run logs? We need those if we want to update the metadata to record intermittent tests correctly.

Reply:
So this was mainly to avoid recording all the logs for when we use `try_notify`. Otherwise the bot will notify with a count of all the tests run including any repeats, which may not be the most useful notification? This is off by default for all other pieces of functionality in wpt-sync.

Reply:
In that case we should probably call it `first_only` or `one_per_job` or something. In practice I think the notification code only works with one log per job name so at most we end up redoing some work. Still it seems like it would be better if we actually tried to use the data to tell people about any instabilities, but agreed that we shouldn't block this change on that.

Reply:
Renaming it sounds like a good idea. I think we should keep this for now, and file an issue about improving the notification system to provide better metrics using these stability runs.